.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy Application to Minikube
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y conntrack
+          make install-python-deps
+
+      - name: Install Minikube
+        run: |
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          sudo install minikube-linux-amd64 /usr/local/bin/minikube
```

Comment:
Why not use a dedicated Minikube install action?

Reply:
Since it would require a third party GitHub Action, which is not allowed by policy.

Reply:
You can request an admin (Tim or Michiel) to add an exception to the list of allowed actions.